portfolio/src/main/webapp/src/com.js

```diff
@@ -98,6 +132,17 @@ function createCommentCard(commentText) {
   cardContents.innerHTML = commentText;
 
   cardElement.appendChild(cardContents);
+
+  if (typeof sentimentScore == 'number') {
```

Comment:
this can be reduced to `if (sentimentScore)`, because of falsiness

Reply:
Will address in this PR.

Reply:
What about `0`?

Reply:
Nevermind. Thanks for catching that before I fix this.

Reply:
good catch! thanks caroline